app/controllers/AdviceStartController.scala

```diff
@@ -38,12 +38,14 @@ class AdviceStartController @Inject() (
 ) extends FrontendBaseController
     with I18nSupport {
 
-  def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    Ok(view())
+  def onPageLoad(recordId: String): Action[AnyContent] = (identify andThen getData andThen requireData) {
+    implicit request =>
+      Ok(view(recordId))
   }
 
   //TODO navigate to correct page
```

Comment:
is todo still valid? if so does it need to be moved to the navigator?

Reply:
think todo is still valid and it has been on onsubmit def, not sure if we need to move it inside the def and above redirect, as onsubmit has only redirect in it nothing else, any thoughts?

Reply:
removed